_includes/releases/v22.2/v22.2.7.md

```diff
@@ -0,0 +1,147 @@
+- Added two views to the [`crdb_internal`](../v22.2/crdb-internal.html) catalog: `crdb_internal.statement_statistics_persisted`, which surfaces data in the persisted system, and `statement_statistics table` and `crdb_internal.transaction_statistics_persisted`, which surfaces the `system.transaction_statistics` table. [#98684][#98684]
+- The SQL Activity page now displays only persisted stats when selecting to [view fingerprints](../v22.2/ui-statements-page.html#statement-fingerprints-view). This means data recently executed might take up to 10 minutes to show on the [DB Console](../v22.2/ui-overview.html). [#99044][#99044]
+
+<h3 id="v22-2-7-operational-changes">Operational changes</h3>
+
+- Added a new [metric](../v22.2/metrics.html) `changefeed.schema_registry.retry_count`. This measures the number of request retries performed when sending requests to the [changefeed schema registry](../v22.2/changefeed-examples.html). Observing a non-zero value may indicate improper configuration of the schema registry or changefeed parameters. [#98349][#98349]
+
+<h3 id="v22-2-7-command-line-changes">Command-line changes</h3>
+
+- [Workloads](../v22.2/cockroach-workload.html) that take a `--seed` argument used to default to `1`. Now, they use a randomly generated seed in each run. Users can still pass a custom seed with the `--seed` flag. [#95695][#95695]
+- The `--drain-wait` argument to the [`cockroach node drain`](../v22.2/cockroach-node.html) command will be automatically increased if the command detects that it is smaller than the sum of the [cluster settings](../v22.2/node-shutdown.html#cluster-settings) `server.shutdown.drain_wait`, `server.shutdown.connection_wait`, `server.shutdown.query_wait` times two, and `server.shutdown.lease_transfer_wait`. If the `--drain-wait argument` is `0`, then no timeout is used. [#98577][#98577]
+
+<h3 id="v22-2-7-db-console-changes">DB Console changes</h3>
+
+- Fixed the error `Cannot read properties of undefined (reading 'length')` which can cause [DB Console](../v22.2/ui-overview.html) pages fail to load. [#98236][#98236]
+- Updated the column selector icon. [#98915][#98915]
+- Updated the Jobs table column name to the correct Last Execution Time. [#99036][#99036]
+
+<h3 id="v22-2-7-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in [v22.1](../releases/v22.1.html) that caused the internal error `no bytes in account to release ...`. [#97773][#97773]
+- Fixed a bug that caused an internal error when trying to execute a [UDF](../v22.2/user-defined-functions.html) with an empty function body. This bug was present since UDFs were introduced in [v22.2.0](../releases/v22.2.html). [#93834][#93834]
```

Comment:
The link to v22.2.0 here is self-referencing the page, perhaps add the correct section? I am assuming (#v22-2-0)?